Dockerfile

```diff
@@ -5,25 +5,27 @@ LABEL maintainer="Alexander Litvinenko <array.shift@yahoo.com>"
 ENV APP_NAME Dockovpn
 ENV APP_INSTALL_PATH /opt/${APP_NAME}
 ENV APP_PERSIST_DIR /opt/${APP_NAME}_data
+ENV NET_ADAPTER eth0
 
 WORKDIR ${APP_INSTALL_PATH}
 
 COPY scripts .
 COPY config ./config
 COPY VERSION ./config
 
-RUN apk add --no-cache openvpn easy-rsa bash netcat-openbsd zip dumb-init && \
+RUN apk add --no-cache python3 py-pip openvpn easy-rsa bash netcat-openbsd zip dumb-init && \
```

Comment:
I thoroughly reviewed the templating part and it's truly amazing! However it brings in another execution environment i.e python and package dependencies which leads to much greater image size. At the moment I'd like to keep resulting image as small as possible for the benefit of embedded and RISK architectures. 
<img width="919" alt="Screenshot 2022-09-05 at 23 22 45" src="https://user-images.githubusercontent.com/4963165/188512976-ce4548c4-f6d2-4c95-8296-8a4514b43590.png">